content/en/docs/home/contribute/content-organization.md

```diff
@@ -1,16 +1,19 @@
 ---
 title: Content Organization
 date: 2018-04-30
+content_template: templates/concept
 weight: 42
 ---
 
-This site uses Hugo. In Hugo, [content organization](https://gohugo.io/content-management/organization/) is a core concept.
+{{< toc >}}
 
-Also see:
+{{% capture overview %}}
 
-* [Custom Hugo Shortcodes](/docs/home/contribute/includes)
-* [Style Guide](/docs/home/contribute/style-guide)
+This site uses Hugo. In Hugo, [content organization](https://gohugo.io/content-management/organization/) is a core concept.
+
+{{% /capture %}}
 
+{{% capture body %}}
 
 {{% note %}}
```

Comment:
While you're in here, can you fix the leading whitespace here? It should be at column 0.

Reply:
Or maybe this is right and the Github UI is playing tricks on my eyes.